### ISSUE TITLE
Map rendering

### ISSUE THREAD

**Issue by ThomasShaped** [pkmn-world/pkmn-world#2]
Here's an issue to track the ongoing map generation task. I assume @bastuijnman you are on this one still? At some point in the future we will need a way of parsing the data that comes back from mapzen too I guess!

**Comment by bastuijnman** [comment]
Yup, working on that, currently parsing mapzen data. Next step is proper tile rendering
